RSSReader/src/app/RSS_GUI.fxml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import java.util.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+
+<AnchorPane id="AnchorPane" prefHeight="537.0" prefWidth="947.0" xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/8">
```

Comment:
We'll need to create and set a controller for this. SceneBuilder has a "show skeleton controller" menu item (or similarly named) that will show you, or you can renamed the FXMLDocument one. You'll need to alter it to match the fxml doc. We can do this in a separate PR when we hook up the controller.